electrumsv/wallet_database/functions.py

```diff
@@ -2411,15 +2411,17 @@ def _write(db: Optional[sqlite3.Connection]=None) -> PasswordUpdateResult:
         assert db is not None and isinstance(db, sqlite3.Connection)
         password_token = pw_encode(os.urandom(32).hex(), new_password)
 
-        cursor = db.execute(token_update_sql, (date_updated, password_token, "password-token"))
+        cursor = db.execute(token_update_sql, (date_updated, json.dumps(password_token),
+            "password-token"))
         assert cursor.rowcount == 1
 
         # This tracks the updated encrypted values for the wallet to replace cached versions of
         # these with.
         result = PasswordUpdateResult(
             password_token=password_token,
             masterkey_updates=[],
-            account_private_key_updates={})
+            account_private_key_updates={},
```

Comment:
Why is this being set now when it wasn't before?

Reply:
I added a field to it to pass the server updates back to the callback but then I removed it... The only net change is the added trailing `,` . But I will remove that so it doesn't show up on the diff.